### ISSUE TITLE
`env` is not recognized as an internal or external command on windows Golang

### ISSUE THREAD

**Issue by rayhaanq** [serverless/serverless#4764]
Hi there, I understand that this is because `env` is a linux command. But following the Golang example on the serverless website how would I run the `make` command successfully? I had to install make from GnuWin32. Also, will it be necessary to update the Makefile for each new function? I've tried on powershell and cmd. Thanks

Serverless Framework Version: 1.26.0
Operating System: Windows 10

Error messages:
```dep ensure
env GOOS=linux go build -ldflags="-s -w" -o bin/hello hello/main.go
'env' is not recognized as an internal or external command,
operable program or batch file.
make: *** [build] Error 1```

**Comment by rayhaanq** [comment]
work around is to use git bash

**Comment by HyperBrain** [comment]
Hi @rayhaanq , it seems that it is not easily possible to invoke a single command with specific environment variables set in PowerShell (see https://stackoverflow.com/questions/1420719/powershell-setting-an-environment-variable-for-a-single-command-only).

For me - I personally use Git Bash for all Serverless tasks we have.

**Comment by rayhaanq** [comment]
Hi @HyperBrain thanks for the response. I created a bat file for it and i think it seemed to deploy fine. However when it's deployed I keep getting an `internal server error` and I can't seem to understand why. I've tried editing the response by filling in all the fields. I've used the code mentioned in the serverless website with no luck. I've tried to use some code that shown on acloudguru too but still no luck. I thought maybe it was because the zip file wasn't being generated properly so I created it manually using the zip generator provided by aws but still didn't work. I also tried installing a linux VM and built then generated the zip file from there. Still the same error. Any ideas what this could be? I've literally been at it none stop... I think the solution is quite simple but I really don't know what else to try. Some help will be much appreciated. I've created an issue which shows the error message here: https://github.com/serverless/serverless/issues/4766
